video/out/cocoa_cb_common.swift

```diff
@@ -170,13 +170,7 @@ class CocoaCB: Common {
     var controlCallback: mp_render_cb_control_fn = { ( v, ctx, e, request, d ) -> Int32 in
         let ccb = unsafeBitCast(ctx, to: CocoaCB.self)
 
-        // the data pointer can be a null pointer, the libmpv control callback
-        // provides nil instead of the 0 address like the usual control call of
-        // an internal vo, workaround to create a null pointer instead of nil
-        var data = UnsafeMutableRawPointer.init(bitPattern: 0).unsafelyUnwrapped
-        if let dunwrapped = d {
-            data = dunwrapped
-        }
+        let data = UnsafeMutableRawPointer(d)
```

Comment:
i believe i tried it like this but there was a reason at the time i didn't use it. i need to look into this.

Reply:
on a side note, i believe `d` is already a `UnsafeMutableRawPointer?`, we don't need to initialise a new one here and could pass it directly to the ccb.control() function.